common/src/stack/command/stack/commands/load/__init__.py

```diff
@@ -1,12 +1,221 @@
-# @rocks@
-# Copyright (c) 2000 - 2010 The Regents of the University of California
-# All rights reserved. Rocks(r) v5.4 www.rocksclusters.org
-# https://github.com/Teradata/stacki/blob/master/LICENSE-ROCKS.txt
-# @rocks@
-#
+# @copyright@
+# Copyright (c) 2006 - 2018 Teradata
+# All rights reserved. Stacki(r) v5.x stacki.com
+# https://github.com/Teradata/stacki/blob/master/LICENSE.txt
+# @copyright@
 
+
+import stack.commands
+import json
+import sys
+import os
+from stack.exception import CommandError, ArgRequired, ArgUnique
+import subprocess
+import logging
 import stack.commands
+from stack.bool import str2bool
+import re
+import shlex
 
 
 class command(stack.commands.Command):
-	pass
+	MustBeRoot = 0
+
+	def _load(self, text):
+		try:
+			data = json.loads(text)
+		except ValueError as e:
+			# parse the error message and split the input at the
+			# syntax error
+			i = int(re.search(r'char (.*?)\)', str(e)).group(1))
+			b = text[:i]
+			a = text[i:]
+
+			# find the line with the error and report it
+			# 'blen' in honor of our intern
+
+			line1 = (b[b.rfind('\n'):] + a[:a.find('\n')]).strip()
+			blen  = len((b[b.rfind('\n'):]).strip())
+			line2 = ' ' * blen + '^'
+			raise CommandError(self,
+					   f'syntax error\n{line1}\n{line2}\n{e}')
+		return data
+
+	def get_scope(self):
```

Comment:
If you want to do get/set methods in a more 'pythonic' way, you should use property decorators, like so:
```
	@property
	def scope(self):
		try:
			return self.__dump_scope
		except AttributeError:
			return 'global'

	@scope.setter
	def scope(self, value):
		self.__dump_scope = value
```

Then instead of doing `self.get_scope()` or `self.set_scope()` you just access it like a member variable `do_foo(self.scope)` or `self.scope = 'foo'`